### DISCUSSION TITLE
TS011F_plug_3 reporting incorrect currents

### DISCUSSION THREAD

**Discussion by DrSamjames** [Koenkk/zigbee2mqtt#15533] [Devices]
Hi

I've got some Zigbeee power monitoring plus off AliExpress. Two of them are reporting as TS011F_plug_1 and two as TS011F_plug_3.

They are mainly working correctly apart from the fact that the TS011F_plug_3's are reporting clearly incorrect currents. I can find lots about switching off issues and the lack of automatic reporting, but abnormal currents seems to be new?

Thanks

Sam

**Comment by rolo64416** [comment]
Same here, have a TS011F_plug_3 that's reporting false value's. I use it for monitoring the washing machine. Worked fine for a year. I have replaced it and will do some testing with the plug.

**Comment by DrSamjames** [comment]
Was it the same plug that started reporting incorrect values or was it the
replacement?

Did you do anything like a firmware update?

Sam

**Reply by rolo64416** [reply]
The first plug. I did not change anything. I noticed it when the "washingmachine ready" notify came over a hour to late. I measure average power over a period of time.

**Comment by rolo64416** [comment]
I have two of these plugs, both report as TS011F_plug_3. I do see a difference in firmware. Plug 1 is the original one that gave the problems. My first tests show that plug 1 does not report low power values ( < 5 W), plug 2 does this correct. I think that the internals are different. This does not explain why it did work before. Tolerances of used parts? 
 
![2022-12-17 10_44_44-Window](https://user-images.githubusercontent.com/61646347/208236180-1d7ad68d-dce1-481b-8fca-614b7a620173.png)

**Reply by janpep** [reply]
![ZBplug-3](https://github.com/user-attachments/assets/2217a669-1a59-44a5-8ff3-8158ee897725)
While searching I came across this older thread, which seems to describe the same thing as what I am currently running into.
I also see incorrect values. Is there a solution?
I now have two days of experience with Sonoff Dongle Plus-E and zigbee2MQTT, where I bought two plugs to experiment with.
Both plugs TS011F presents itself as TS011F_plug_3. The first one seems to work correct. The other shows fluctuating values from 0 to 12 Watts, **while nothing is connected**. And when something is connected, these are falsely high values. These values are also passed on very frequently to Domoticz. Al least once per second.
- What can I do to get the correct values once per minute? OR
- How do I possibly disable the passing on of these incorrect values.
